pkg/config/kubelet.go

```diff
@@ -60,6 +61,9 @@ featureGates:
   ServiceNodeExclusion: true
   SupportPodPidsLimit: true
 serverTLSBootstrap: false #TODO`)
+	if _, err := os.Stat("/run/systemd/resolve/resolv.conf"); !errors.Is(err, os.ErrNotExist) {
```

Comment:
please add a comment and reference the coredns recommendation

Reply:
Added! Thanks for pointing this out.